actors/common/src/main/java/org/sunbird/metrics/actors/CourseMetricsActor.java

```diff
@@ -74,7 +75,8 @@ public void onReceive(Request request) throws Throwable {
 
   private void courseProgressMetricsReport(Request actorMessage) {
 
-    ProjectLogger.log("CourseMetricsActor-courseProgressMetrics called");
+    ProjectLogger.log(
+        "CourseMetricsActor:courseProgressMetricsReport call start", LoggerEnum.INFO.name());
```

Comment:
From:
"CourseMetricsActor:courseProgressMetricsReport call start"

To:
"CourseMetricsActor: courseProgressMetricsReport called"